server/api/controllers/class/index.js

```diff
@@ -2,46 +2,79 @@ const ModelsRoot = require('../../models');
 
 let Models = null;
 let db = null;
-let container = {};
+let classContainer = {};
 
-container.readOne = async (id) => {
+// TODO:PUSH:
+classContainer.readOne = async (id) => {
   try {
-    return await Models.class.findOne({
+    let record = await Models.class.findOne({
       where: { id: id }
     });
+
+    if (record instanceof Error) throw record;
+
```

Comment:
I am not a fan of this throw request.  I believe it should be configured with something like "Error Fetching Record From Database for Record #????".

That would be my guess.